tutorial/source/index.rst

```diff
@@ -3,92 +3,202 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to Pyro Examples and Tutorials!
-==========================================
+Getting Started With Pyro: Tutorials, How-to Guides and Examples
+================================================================
+
+Welcome! This page collects tutorials written by the Pyro community.
+If you're having trouble finding or understanding anything here,
+please don't hesitate to ask a question on our `forum <https://forum.pyro.ai/>`_!
+
+New users: getting from zero to one
+------------------------------------
+If you're new to probabilistic programming or variational inference,
+you might want to start by reading the series :ref:`introductory-tutorials`.
+
+After that, you're ready to get your hands dirty! (Yes, really!)
```

Comment:
should we avoid English-specific idioms so as to attract more non-native English readers?